CHANGELOG.md

```diff
@@ -10,9 +10,13 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 - Fix Edm.Guid mutator 
 - Fix extra single quote in URL for datetime
-- Pinned install dependencies for deterministic installations
+- Pin install dependencies for deterministic installations
 - Use the new signal handler interface after update to latest Gevent library 20.5.0
 
+### Added
+
+- Print error messages, which cause the fuzzer to stop, to stdout
```

Comment:
I am still considering this as bugfix of incorrect logging than whole new feature. That would be anything marked as enhancement in issues.

Reply:
I consider this to be a feature since the logging was already present there.

Reply:
If it was there.. then it is a fix of something that is there :)